src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifFileWithContentsTests.cs

```diff
@@ -16,9 +16,15 @@ namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
     [Collection("SarifObjectTests")]
     public class SarifFileWithContentsTests
     {
-        private const string Key = "/item.cpp#fragment";
-        private const string Key2 = "/item2.cpp";
-        private const string Contents = "This is a test file.";
+        private const string key1 = "/item.cpp#fragment";
```

Comment:
`PascalCase` for `const` (_e.g._, `Key1`).

Reply:
Oh right, fixed